CODEOWNERS

```diff
@@ -150,6 +150,12 @@ k8s/**/common/family-public-law/namespace.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/identity.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/fpl-case-service.yaml @hmcts/fpl-admins
 
+### Adoption
+k8s/**/common/adoption/namespace.yaml @hmcts/fpl-admins
+k8s/**/common/adoption/identity.yaml @hmcts/fpl-admins
+k8s/**/common/adoption/adoption-frontend.yaml @hmcts/fpl-admins
```

Comment:
This isn't allowed as this allows you to go live with these apps on prod without approvals.

Reply:
narrowed to non prod envs